package.json

```diff
@@ -15,6 +15,7 @@
     "@types/long": "^4.0.0",
     "abort-controller": "^3.0.0",
     "duplexify": "^4.0.0",
+    "extend": "^3.0.2",
```

Comment:
Can we avoid pulling one more dependency? Any external dependency is a liability. It's only used once, when setting values from `DEFAULTS`, and since `DEFAULTS` does not have any nested structures, it can be done using a simple iteration over `Object.keys(DEFAULTS)`.

Reply:
Sure! For context, I think it's technically already being pulled in implicitly as a dependency of retry-request and since this logic was pulled directly from retry-request we then had to explicitly add it. Happy to remove it though because it's not crucial!

Reply:
Update, based on the [docs for extend](https://www.npmjs.com/package/extend) it looks like a one liner with Object.assign! 😁 🥳